pyLib/optics.py

```diff
@@ -0,0 +1,496 @@
+"""
+This a basic python module for managing the lattice and optics, e.g.
+   inspecting tune, chroma, twiss functions, phase spaces, etc...
+"""
+
+
+__version__ = '0.0'
+__author__ = 'Angelos Efstratiou and Alessio Mereghetti'
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+
+
+def plot_beta_dx(myLine, plot_x=False):
```

Comment:
Maybe a possible improvement to the function could be:
* to make a twiss plot [as available in `xplt`](https://xsuite.github.io/xplt/latest/examples/twiss.html#default-twiss-plot);
* if the user adds normalised emittances, to make a beam-envelope and accelerator-aperture plot [as available in `xplt`](https://xsuite.github.io/xplt/latest/examples/twiss.html#apertures).

Then, the function interface could have the Xsuite line object (mandatory), and the normalised emittances (optional); if the latter are given, the aperture-envelope plot is created automatically, otherwise the function plots only the twiss part.